src/TLabel.cpp

```diff
@@ -82,6 +82,13 @@ void TLabel::mousePressEvent(QMouseEvent* event)
     QWidget::mousePressEvent(event);
 }
 
+void TLabel::mouseDoubleClickEvent(QMouseEvent* event)
+{
+    if (forwardEventToMapper(event)) {
```

Comment:
Technically - as this class does not do anything with double clicks you do not need to have anything here other than:

    forwardEventToMapper(event);

depending on the level of warnings specified to your compiler you _may_ get a warning about an unused return value - but then I am sure we do the same sort of thing (defined a function to return a value and then not use it) elsewhere, I guess [Coverity](https://www.coverity.com) will moan - so we may need to look at formalising a way to suppress such thing - I found [this](https://stackoverflow.com/q/689677/4805858) on SE that expands on the subject.

Reply:
Otherwise I think this is good to go :shipit: